extensions/default/src/hangingprotocols/viewportOptions.ts

```diff
@@ -0,0 +1,18 @@
+/** A default viewport options */
+export const viewportOptions = {
```

Comment:
I think we should move this and other selectors to a utils folder to keep it organized and clarify that it is not a hanging protocol, similar to other files in this path.

Reply:
Maybe we also call it `defaultViewportOptions`?